wiki/Ranking_Criteria/Difficulty_Naming/en.md

```diff
@@ -1,6 +1,6 @@
 # Difficulty Naming
 	
-The purpose of difficulty naming is to inform players how difficult a map is. This article lists common conventions as well as other alternatives that have been gathered throughout the years. However, as long as the set of difficulty names is descriptive and clearly indicates progression of difficulty, you may use any desired kind of scheme.
+The purpose of difficulty naming is to inform players how difficult a map is. This article lists common conventions as well as other alternatives that have been gathered throughout the years. However, as long as the set of difficulty names is descriptive and clearly indicates progression of difficulty, you may use any desired kind of scheme. Often times, the highest difficulty of a mapset will use a special thematic name.
```

Comment:
probably revert because this sentence doesn't add anything to the rules being enforced by RC. it's also already covered in main RC, I think